CRVResponse/src/CrvPhotonGenerator_module.cc

```diff
@@ -312,8 +312,9 @@ namespace mu2e
             if(t1Tmp<eventWindowStart) t1Tmp+=_microBunchPeriod;
             if(t2Tmp<eventWindowStart) t2Tmp+=_microBunchPeriod;
             //remove steps that are not needed to speed up the simulation
-            if(t1Tmp<startTime) continue;
-            if(t2Tmp>endTime) continue;
+            if(t1Tmp<startTime && t2Tmp<startTime) continue;
```

Comment:
Is this to deal with simulated data wrapping or real data?

Reply:
That fixes a minor issue in the simulation, where I previously ignored some CrvSteps around the edges of the live window that shouldn't be ignored. The impact is very small.